src/main/java/frc/robot/Robot.java

```diff
@@ -108,13 +110,48 @@ public void robotInit() {
                 () -> -controller.getLeftX() * SwerveSubsystem.MAX_LINEAR_SPEED));
 
     NamedCommands.registerCommand("stop", swerve.stopWithXCmd().asProxy());
+
+    controller.b().toggleOnTrue(autoAimDemo(() -> swerve.getVelocity()));
   }
 
   @Override
   public void robotPeriodic() {
     CommandScheduler.getInstance().run();
   }
 
+  public Command autoAimDemo(Supplier<ChassisSpeeds> speeds) {
+
+    return Commands.sequence(
```

Comment:
add some comments to this to explain how it works, and a doc comment to the method